src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidationError.swift

```diff
@@ -25,33 +25,59 @@ enum TicketValidationError: LocalizedError {
 		switch self {
 		case .validationDecoratorDocument(let error):
 			switch error {
-			case .REST_SERVICE_ERROR(.receivedResourceError(.VD_ID_CLIENT_ERR)), .VD_ID_EMPTY_X5C, .VD_ID_NO_ATS_SIGN_KEY, .VD_ID_NO_ATS_SVC_KEY, .VD_ID_NO_ATS, .VD_ID_NO_VS_SVC_KEY, .VD_ID_NO_VS, .REST_SERVICE_ERROR(.receivedResourceError(.VD_ID_PARSE_ERR)):
+			case .VD_ID_EMPTY_X5C, .VD_ID_NO_ATS_SIGN_KEY, .VD_ID_NO_ATS_SVC_KEY, .VD_ID_NO_ATS, .VD_ID_NO_VS_SVC_KEY, .VD_ID_NO_VS:
```

Comment:
Fells like this should not be in this PR?

Reply:
Yeah reverted it.